core/src/main/scala/kanaloa/reactive/dispatcher/queue/Worker.scala

```diff
@@ -148,16 +147,17 @@ private[queue] class Worker(
 
   private def retryOrAbandon(outstanding: Outstanding, error: Any): Unit = {
     outstanding.cancel()
+    val errorDesc = descriptionOf(error, outstanding.work.settings.lengthOfDisplayForMessage)
     if (outstanding.retried < outstanding.work.settings.retry) {
-      log.debug(s"Retry work $outstanding")
+      log.info(s"Retry work $outstanding due to Error $errorDesc")
```

Comment:
I'm guessing `Error` isn't meant to be capitalized here (the string below has it as lowercase)